apps/omg_utils/mix.exs

```diff
@@ -25,5 +25,5 @@ defmodule Utils.MixProject do
   defp elixirc_paths(:prod), do: ["lib"]
   defp elixirc_paths(_), do: ["lib", "test/support"]
 
-  defp deps, do: [{:ecto_sql, "~> 3.1"}]
+  defp deps, do: [{:ecto_sql, "~> 3.1"}, {:appsignal, "~> 1.9"}]
```

Comment:
why is ecto_sql included again?

Reply:
I worked on the older version of the master branch, 
I am already doing rebase